lightning-background-processor/src/lib.rs

```diff
@@ -91,20 +97,14 @@ where
 	///
```

Comment:
(on the next line up, but Github is dumb) not your bug, but it looks like there's a missing `)` there, can you add it?